### PR TITLE
:sparkles: Pool Details Header

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206685046725483

### CODE REVIEW

File: views/pool-details/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <Layout>
+      <PoolTitleBar
+        iconTokenList={tokens.map(
+          (token) => TOKEN_ICONS[network][token.symbol]
+        )}
+        onBack={() => push(Routes[RoutesEnum.Pools])}
+        name={tokens.reduce(
+          (acc, token) =>
+            acc === '' ? token.symbol : acc + '•' + token.symbol,
```
Comment (git-marcopitra) [inline views/pool-details/index.tsx:35]: Instead `+` concat u must use template string

File: views/pools/pool-card/index.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ const PoolCard: FC<PoolCardProps> = ({ token0, token1, dex, poolObjectId }) => {
         dexUrl={DEX_MAP[dex].url}
         Logo={<Icon width="100%" maxWidth="1.4rem" maxHeight="1.4rem" />}
       />
-      <PoolCardInfo apr="333.45" coins={[token0, token1]} />
+      <PoolCardInfo apr="333.45" coins={tokens} />
```
Comment (git-marcopitra) [inline views/pools/pool-card/index.tsx:40]: add this apr to a constant and reuse it inside

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: constants/pools.ts [hunk 3]
```diff
@@ -11,27 +10,27 @@ export const RECOMMENDED_POOLS: Record<
...
     {
       dex: 'interest',
-      token0: COINS[0],
-      token1: COINS[1],
-      poolObjectId: `${ZERO_BIG_NUMBER}`,
+      tokens: [COINS[0], COINS[1], COINS[2]],
+      poolObjectId: COINS[0].type,
       lpCoin: COINS[0],
       stable: false,
+      apr: '3333.45',
```
Comment (git-marcopitra) [inline constants/pools.ts:17]: remove it from here, cuz apr is a calculated value

File: views/pool-details/index.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <Layout>
+      <PoolTitleBar
+        iconTokenList={tokens.map(
+          (token) => TOKEN_ICONS[network][token.symbol]
+        )}
+        onBack={() => push(Routes[RoutesEnum.Pools])}
+        name={tokens.reduce(
+          (acc, token) =>
+            acc === '' ? token.symbol : `${acc} • ${token.symbol}`,
```
Comment (git-marcopitra) [inline views/pool-details/index.tsx:35]: Simplify to:

```ts
`${acc ? `${acc} • ` :  ''}${token.symbol}`
```

File: views/pools/pool-card/index.tsx [hunk 5]
```diff
@@ -8,7 +8,7 @@ import PoolCardHeader from './pool-card-header';
 import PoolCardInfo from './pool-card-info';
 import PoolCardTrade from './pool-card-trade';
 
-const PoolCard: FC<PoolCardProps> = ({ token0, token1, dex, poolObjectId }) => {
+const PoolCard: FC<PoolCardProps> = ({ tokens, dex, apr, poolObjectId }) => {
   const Icon = DEX_MAP[dex].Icon;
```
Comment (git-marcopitra) [inline views/pools/pool-card/index.tsx:13]: `apr` should be a constant here to be calculated later on fetch the values

File: views/pools/pool-card/pool-card.types.ts [hunk 6]
```diff
@@ -24,8 +24,8 @@ export interface PoolCardTradeProps {
 export interface PoolCardProps {
   dex: string;
   stable: boolean;
-  token0: CoinData;
-  token1: CoinData;
+  tokens: ReadonlyArray<CoinData>;
   lpCoin: CoinData;
   poolObjectId: string;
+  apr: string;
```
Comment (git-marcopitra) [inline views/pools/pool-card/pool-card.types.ts:30]: u can remove it from here

**Review by git-marcopitra — APPROVED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Pull request merged**